drivers/include/periph/pwm.h

```diff
@@ -85,6 +85,31 @@ extern "C" {
 #define PWM_UNDEF           (UINT_MAX)
 #endif
 
+/**
+ * @brief  Default number of onboard PWM devices
+ */
+#ifndef PWM_NUMOF
+#define PWM_NUMOF           (0U)
+#endif
+
+/**
+ * @brief   Default number of PWM extension devices
+ */
+#ifndef PWM_EXT_NUMOF
```

Comment:
Ok, I just placed it also in `drivers/include/periph/pwm.h` to avoid wrong definitions if someone includes `drivers/include/periph/pwm.h` before `drivers/include/extend/pwm.h`.